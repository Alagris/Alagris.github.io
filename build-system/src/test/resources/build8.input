:eval function1 '01'
:eval function1 ''
:eval function2 'this function has no type, hence it defaults to using .* as alphabet'
:eval mirror 'reflect a input'
:eval mirror 'reflect b input'
:eval mirror 'reflect z input'
:eval multiple_types 'aaa'
:eval with_integers <1 3>
:eval with_integers <43 1243>
:eval with_integers <43 1243 43 1243 43 1243>