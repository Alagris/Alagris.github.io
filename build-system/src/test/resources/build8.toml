projectName = "project7"
[[source]]
path = "project7.zip"
