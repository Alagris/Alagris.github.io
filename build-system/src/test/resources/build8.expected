Transducer 'function1' not found!
Transducer 'function1' not found!
Transducer 'function2' not found!
Transducer 'mirror' not found!
Transducer 'mirror' not found!
Transducer 'mirror' not found!
''
''
''
''
